### DISCUSSION TITLE
Example of how to use a server-side HTTP backend with v4?

### DISCUSSION THREAD

**Discussion by brattonross** [sergiodxa/remix-i18next#58] [Q&A] [answered]
I'm looking to migrate my app from v3 to v4, but I'm having trouble understanding how to use a fetch/http backend on the server-side.

In v3 I used the `FetchBackend`, and I'm assuming that in v4 I need to use `i18next-http-backend`, but attempting to follow along with the example in the readme lands me with an error.

Here's what I'm trying:

```ts
// i18n.server.ts
import Backend from 'i18next-http-backend'
import { RemixI18Next } from 'remix-i18next';

export const i18n = new RemixI18Next({
  backend: Backend, // Property 'type' is missing in type 'typeof I18NextHttpBackend' but required in type 'NewableModule<BackendModule<unknown>>'.
});
```

I would assume that this error is happening because there is no `static type: 'backend'` on the HTTP backend class, whereas there is on the fs one.

Please could you provide an example of how you might implement an HTTP backend on the server-side? Thanks

**Comment by sergiodxa** [comment, marked as answer]
If the backend doesn't conform to the interface i18next exports you should open an issue on that backend repository.

The [i18next-fetch-backend](https://github.com/dotcore64/i18next-fetch-backend) has the type key and because Remix polyfill Fetch it should work correctly.

**Reply by brattonross** [reply]
Thank you for the prompt response and recommendations, I'll give `i18next-fetch-backend` a try as well. Cheers!
